client/homebrew/navbar/help.navitem.jsx

```diff
@@ -10,7 +10,7 @@ module.exports = function(props){
 			need help?
 		</Nav.item>
 		<Nav.item color='red' icon='fas fa-fw fa-bug'
-			href={`https://www.reddit.com/r/homebrewery/submit?selftext=true&title=${encodeURIComponent('[Issue] Describe Your Issue Here')}`}
+			href={`https://www.reddit.com/r/homebrewery/submit?selftext=true&text=Browser%28s%29%3A++%0D%0AOperating+System%3A++%0D%0ALegacy+or+v3+renderer%3F%3A++%0D%0AIssue%3A++`}
```

Comment:
can we use `encodeURIComponent` to make this more legible?

Reply:
Yeah, I wasn't sure how to get the line breaks in there and wasn't 100% confident on dedent so skipped it.  But I should have at least tried rather than use an online URL encoder.  

Will back with that in a bit (hopefully).

Reply:
Okay this is done with b1932dc